vs_utils/utils/nnscore_utils.py

```diff
@@ -38,8 +39,44 @@ def force_partial_charge_computation(mol):
   for obatom in openbabel.OBMolAtomIter(mol):
     obatom.GetPartialCharge()
 
-
-def hydrogenate_and_compute_partial_charges(input_file, input_format, output_directory):
+def pdbqt_to_pdb(input_file, output_directory):
+  """Convert pdbqt file to pdb file.
+  
+  Parameters
+  ----------
+  input_file: String
+    Path to input file.
+  output_directory: String
+    Path to desired output directory.
+  """
+  basename = os.path.basename(input_file).split(".")[0]
+  pdb_output = os.path.join(output_directory, basename + ".pdb")
+  print "About to write to"
+  print pdb_output
+  with open(pdb_output, "wb") as outfile:
+    obabel_command = ["obabel", "-ipdbqt", input_file, "-opdb"]
+    subprocess.Popen(obabel_command, stdout=outfile).wait()
+ 
+
+  #conversion = openbabel.OBConversion()
```

Comment:
Could you remove these commented out lines (I know this is my left-over cruft...)